src/evaluate/expression.js

```diff
@@ -225,6 +225,8 @@ function applyBinaryOperation (operator, x, y) {
     case '>': return x > y ? 1 : 0
     case '<=': return x <= y ? 1 : 0
     case '<': return x < y ? 1 : 0
+    case 'and': return x && y ? 1 : 0
+    case 'or': return x || y ? 1 : 0

```

Comment:
`xor` might be a good candidate to include in this PR, since it follows the same pattern as these two (unlike `not(`). Up to you!

Reply:
xor is added!